packages/core/security/core-security-server-internal/src/utils/default_implementation.ts

```diff
@@ -13,5 +13,14 @@ export const getDefaultSecurityImplementation = (): CoreSecurityDelegateContract
     authc: {
       getCurrentUser: () => null,
     },
+    audit: {
```

Comment:
If the Security API is not registered, `core.security.audit.asScoped` and `core.security.audit.withoutRequest` do not implement audit logging.

Reply:
Related to my previous question, why do we need to return `null` instead of not returning anything as written in type definition?

Reply:
Great catch! I've updated the default implementation and unit test.